src/libcollections/str.rs

```diff
@@ -680,38 +834,57 @@ impl str {
         Utf16Units { encoder: Utf16Encoder::new(self[..].chars()) }
     }
 
-    /// Returns `true` if `self` contains another `&str`.
+    /// Returns `true` if the given `&str` is a sub-slice of this string slice.
+    ///
+    /// Returns `false` if it's not.
```

Comment:
Good documentation doesn't rely on implications, so I put it in. Explicit > implicit.